website/pages/en/billing.mdx

```diff
@@ -2,104 +2,84 @@
-
-6. Click 'Add GRT to account balance' to add the GRT to your account balance. Sign the associated transaction in your wallet. This will cost gas.
-
-7. Once the transaction is confirmed, you'll see the GRT added to your account balance within an hour.
-
-### Withdrawing GRT using a crypto wallet
-
-> This section is written assuming you have deposited GRT into your account balance on [Subgraph Studio](https://thegraph.com/studio/billing/) and that you're on the Arbitrum network.
-
-1. Go to the [Subgraph Studio Billing page](https://thegraph.com/studio/billing/).
-
+3. Select the "Manage" button near the top right corner. First time users will see an option to "Upgrade to Growth plan" while returning users will click "Deposit from wallet".
+4. Use the slider to estimate the number of queries you expect to make on a monthly basis.
+   - For suggestions on the number of queries you may use, see our **Frequently Asked Questions** page.
+5. Choose "Cryptocurrency". GRT is currently the only cryptocurrency accepted on The Graph Network.
+6. Select the number of months you would like to prepay.
+   - Paying in advance does not commit you to future usage. You will only be charged for what you use and you can withdraw your balance at any time.
+7. Pick the network from which you are depositing your GRT. GRT on Arbitrum or Ethereum are both acceptable.
+8. Click "Allow GRT Access" and then specify the amount of GRT that can be taken from you wallet.
+   - If you are prepaying for multiple months, you must allow access to the amount that corresponds with that amount. This interaction will not cost any gas.
+9. Lastly, click on "Add GRT to Billing Balance". This transaction will require ETH on Arbitrum to cover the gas costs.
+
+- Note that GRT deposited from Arbitrum will process within a few moments while GRT deposited from Ethereum will take approximately 15-20 minutes to process. Once the transaction is confirmed, you'll see the GRT added to your account balance.
+
+### Withdrawing GRT using a wallet
+
+1. Go to the [Subgraph Studio Billing page](https://thegraph.com/studio/billing/).
 2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect".
-
-3. Click the dropdown next to the 'Add GRT' button at the center of the page. Select withdraw GRT. A side panel will appear.
-
+3. Click the "Manage"" button at the top right corner of the page. Select "Withdraw GRT". A side panel will appear.
 4. Enter the amount of GRT you would like to withdraw.
-
 5. Click 'Withdraw GRT' to withdraw the GRT from your account balance. Sign the associated transaction in your wallet. This will cost gas. The GRT will be sent to your Arbitrum wallet.
-
 6. Once the transaction is confirmed, you'll see the GRT withdrawn from your account balance in your Arbitrum wallet.
 
 ### Adding GRT using a multisig wallet
```

Comment:
Is this process different except that you need to connect a multisig?

Reply:
Removed that, thanks for flagging!